src/index.md

```diff
@@ -242,7 +242,7 @@ options: `article`, `product`
 default: `website`
 
 ### url
-The content for `og:url` and `twitter:url`.
+The content for `<link rel=canonical href='URL'>`, `og:url` and `twitter:url`.
```

Comment:
Can you change `<link rel=canonical href='URL'>` to `<link rel="canonical" href="foo.com">` 

Once that is updated, this looks good to me.

Reply:
done